### DISCUSSION TITLE
R&D & Summary About Traffic-Ai Research Papers

### DISCUSSION THREAD

**Discussion by hossainalmahdi** [Saturn-AI/computer-vision#27] [General]
As there is a lot of work going on Traffic-Ai. I've studied some of the research papers and wrote summary of them. However, As I was talking earlier that; detecting/predicting traffic collision from a TPP view is more complex than fpv view. For example, we have to deal with more parameters. camera setup is crucial, we have to handle false prediction/detection and etc. Moreover,  detecting Traffic congestion is also a tough task if we compare with real life implementation.

Although I've my own perspective. But I would like to hear from the people of this organization. Here is the link of some research papers that I've studied and the summaries are attached to it.
Link:  https://drive.google.com/drive/folders/1g0_NYcHp_3tQEGXRDfMJDrgPrLq19BcL?usp=sharing

**Comment by bhuiyanmobasshir94** [comment]
@Saturn-AI/ml-team
